README.md

```diff
@@ -2,124 +2,118 @@
+- [ ] Release a stable gitbase version
+  - [ ] Add regression testing
+- [ ] Ensure gitbase interoperability with 3rd party tools
 
-- [ ] Improve gitbase performance
-- [ ] Release a more stable gitbase version
-    - [ ] Make regression test
-- [ ] Engine using gitbase as backend
-    - [ ] Change engine schema to match gitbase's
-    - [ ] Replace actual JGit implementation with a query builder and calls to gitbase
-- [ ] [P1] gitbase interoperability with 3rd party tools
+#### Improve gitbase performance to single second responses on most queries 
+- [ ] **Key results pending an internal engineering meeting**
+- [ ] Switch Engine to use gitbase as a backend
+  - [ ] Change engine schema to match gitbase's
+  - [ ] Replace actual JGit implementation with a query builder and calls to gitbase
 
-### Babelfish
+### Setup a stable 'Code As Data' deployment in our cluster for BigCompanyG
 
-- [ ] Client/Library improvements, packaging, and stability
-- [ ] Iterate the higher level UAST abstractions to fix and maintain
+### Release necessary language support for enterprise
 - [ ] C# driver
 - [ ] C/C++ driver
 
-### Gemini
+### Ensure stability of Babelfish in a production environment
+- [ ] Client/Library improvements, packaging, and stability
 
+### [P1] Get Gemini enterprise ready
 - [ ] Release a new version with the integration of function similarity
 
+
 ## Being a better company to work at
 
-- [ ] Setup the ML cluster environment
+### Successfully hire for each open Role
+- [ ] Hire a VP of Engineering
+- [ ] Develop a candidate pipeline for VP of Sales
+- [ ] Improve candidate pipeline on Infrastructure Role
+- [ ] Attract relevant profiles of applicants through our community
+- [ ] Improve the speed of the hiring process from our side
+- [ ] Increase candidate’s motivation and engagement at every interviewing step
+
+#### Having a more diverse funnel in terms of gender and race
+- [ ] Explicit developer community promotion via social media
+- [ ] Proactively reaching out to diverse communities
+- [ ] Proper updates to the website & guide to promote multi-national and cultural environment
+
+#### Make our interview process more remote friendly
+- [ ] Identify bottlenecks in the hiring process that makes it less remote-friendly and solve them
+
+### Improve general happiness and communication
+- [ ] Conduct a happiness survey and react to the unveiled issues
 - [ ] Conduct an engineering survey to improve development practices for Q3 and Q4
 - [ ] Conduct empathy sessions for cross-team documentation validation
-- [ ] Regression Test, Benchmarks and Deploy platform (CD?)
-    - For which projects? Are any of ML included? - Vadim
+- [ ] Improve internal communication for events happening at source{d}
+- [ ] Improve the formal process of speaking at a conference
+- [ ] Resolve connectivity problems at the Madrid office
+
+#### Setup GPU production cluster for ML
+- [ ] Research and install Kubeflow (or alternatives) to the production cluster
```

Comment:
Unsure about who's going to be in charge of this. It is highly unlikely that infra will have bandwidth for this

Reply:
@rporres relevant changes in https://github.com/src-d/okrs/pull/78